packages/beacon-node/src/chain/chain.ts

```diff
@@ -134,7 +134,8 @@ export class BeaconChain implements IBeaconChain {
   readonly checkpointBalancesCache: CheckpointBalancesCache;
   // TODO DENEB: Prune data structure every time period, for both old entries
   /** Map keyed by executionPayload.blockHash of the block for those blobs */
-  readonly producedBlobsSidecarCache = new Map<RootHex, deneb.BlobsSidecar>();
+  readonly producedBlobSidecarsCache = new Map<RootHex, {blobSidecars: deneb.BlobSidecars; slot: Slot}>();
```

Comment:
maybe define a new type of `BlockHash` as we use execution block hash here, having `RootHex` is a bit confuse even through Typescript compiles ok